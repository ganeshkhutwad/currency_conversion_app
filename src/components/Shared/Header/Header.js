import React from 'react';
import styles from './Header.module.css';
import PropTypes from 'prop-types';

const Header = ({ title }) => <h3 className={styles.HeaderText}>{title}</h3>;

export default Header;

Header.propTypes = { title: PropTypes.string.isRequired };
