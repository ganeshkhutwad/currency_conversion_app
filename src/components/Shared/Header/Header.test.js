import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';

test('Snapshot For Header Component', () => {
    const { asFragment } = render(<Header title="Test Header" />);
    expect(asFragment()).toMatchSnapshot();
});
