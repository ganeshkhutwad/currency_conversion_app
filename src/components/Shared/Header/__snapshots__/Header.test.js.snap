// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot For Header Component 1`] = `
<DocumentFragment>
  <h3
    class="HeaderText"
  >
    Test Header
  </h3>
</DocumentFragment>
`;
